etc/kayobe/ansible/templates/stackhpc-secrets.yml.j2

```diff
@@ -0,0 +1,3 @@
+---
+# SASL password when using libvirt on the host
+secrets_libvirt_sasl_password: "{{ secrets_libvirt_sasl_password | default(lookup('password', '/dev/null'), true) }}"
```

Comment:
Could you not use the password lookup plugin, and write the result to a file?

Reply:
Could work. Are you imagining to just use that for the AIO? Or would we write it some location in kayobe config? Something like `{{ kayobe_config_env_path }}/secrets/libvirt_sasl_password`?